tools/cell_census_builder/tests/test_release_manifest.py

```diff
@@ -0,0 +1,132 @@
+import datetime
+from typing import Type, cast
+
+import pytest
+from cell_census_builder.build_state import CENSUS_CONFIG_DEFAULTS
+from cell_census_builder.release_manifest import (
+    CELL_CENSUS_REGION,
+    CensusDirectory,
+    CensusLocator,
+    CensusVersionDescription,
+    CensusVersionName,
+    get_release_manifest,
+    validate_release_manifest,
+)
+
+from .conftest import has_aws_credentials
+
+# Should not be a real URL
+TEST_CENSUS_BASE_URL = "s3://bucket/path/"
+
+
+@pytest.mark.live_corpus
+def test_get_release_manifest() -> None:
+    census_base_url = CENSUS_CONFIG_DEFAULTS["cell_census_S3_path"]
+    release_manifest = get_release_manifest(census_base_url, s3_anon=True)
+    assert len(release_manifest) > 0
+    assert "latest" in release_manifest
+    assert release_manifest["latest"] in release_manifest
+    validate_release_manifest(census_base_url, release_manifest, s3_anon=True)
+
+
+@pytest.mark.skipif(not has_aws_credentials(), reason="Unable to run without AWS credentials")
```

Comment:
should we fail instead of skip? (in what cases would we not have credentials when running tests?)

Reply:
There are no credentials when run in GHA (that I am aware of)

Reply:
We can add them very easily now, if desired.

Reply:
I am going either way - it is not a very important test.  Perhaps as follow-up work as I'd like to land this PR before I am gone.